packages/button-theme/__tests__/DatePicker.test.tsx

```diff
@@ -8,6 +8,11 @@ import 'regenerator-runtime/runtime';
 expect.extend(toHaveNoViolations);
 
 describe('DatePicker', () => {
+  beforeAll(() => {
+    // TODO: Remove this once https://github.com/nickcolley/jest-axe/issues/147 is fixed.
```

Comment:
Could we add a tech debt card for this too?

Reply:
Tech debt card created [here](https://github.com/button-inc/service-development-toolkit/issues/493)